examples/Makefile

```diff
@@ -1,6 +1,8 @@
+CPPFLAGS ?=
```

Comment:
Where exactly are CPPFLAGS used in liburing?
AFAIR It doesn't depend on anything to propagate them.

Reply:
They are not currently used, that's the point of this patch. :)